composer.json

```diff
@@ -9,8 +9,7 @@
     "require": {
         "php": ">=5.3.3",
         "symfony/symfony": "2.3.*",
-        "doctrine/orm": "~2.2,>=2.2.3,<2.5",
-        "doctrine/dbal": "<2.5",
+        "doctrine/orm": "~2.2.3||~2.3.0||~2.4.8||~2.5",
```

Comment:
changed. I also used the caret operator now. I guess it's time. And it's the responsibility of the people manually updating their dependencies anyway.